test/components/views/messages/MPollBody-test.tsx

```diff
@@ -577,9 +591,9 @@ describe("MPollBody", () => {
 
     it("shows non-radio buttons if the poll is ended", () => {
         const events = [endEvent()];
-        const body = newMPollBody([], events);
-        expect(body.find(".mx_StyledRadioButton")).toHaveLength(0);
-        expect(body.find('input[type="radio"]')).toHaveLength(0);
+        const { container } = newMPollBody([], events);
+        expect(container.querySelectorAll(".mx_StyledRadioButton")).toHaveLength(0);
```

Comment:
Believe we could have a more readable assertion here

```suggestion
        expect(container.querySelectorAll(".mx_StyledRadioButton")).not.toBeInTheDocument();
```